### PR TITLE
speaker & attendees

### LINKED ISSUE #37: Attendees & Speakers introduction section
**Is your feature request related to a problem? Please describe.**
No, its the initial website building part.

**Describe the solution you'd like**
Attendees and speaker glimpse section:
* The section is half build in the figma file as displayed under.
image

* The whole section inspiration is taken from https://www.experiencewelcome.com/
* This section would be the breif of whole technical fest and what all would be happening in it.
* Try to accumulate the animation with images as close as possible and make this section exact replica of https://www.experiencewelcome.com/ section.

**Describe alternatives you've considered**
Discuss the design alternatives with designers and project maintainers.

**Additional context**
Design at https://www.figma.com/file/4a8ZglxR98YOEQpmXu93Ng/Dhruva-team-library?node-id=0%3A1

### PR DESCRIPTION
**Pull request by RaiLokesh**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# Browsers Tested on?

- [x] Chrome
- [x] Firefox
- [x] Edge

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have checked the responsiveness of the component I built
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings

### CODE REVIEW

File: components/AttendeesSpeakers/attendes.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+                    <div className={styles.text}>
+                        <h4 className={styles.h4}>Attendees</h4>
+                        <p className={styles.bl}>No Zoom fatigue here. Optimized to create immersive experiences for attendees, Welcome captivates your audience like never before. Interact with attendees by pulling them on stage, allow them to move freely in and out of rooms, provide networking opportunities, and more.</p>
+                    </div>
+                    <div data-aos="fade-left" className={styles.img}>
+                        <img data-aos="rotate-c" src="https://assets-global.website-files.com/5f8f45cead04af2f54abc938/611fc7ce31ecb97d9ed140ab_use_attendee.jpg" className={styles.hi}/>
+                    </div>
+                    <div className={styles.text}>
+                        <h4 className={styles.h4}>Speakers</h4>
+                        <p className={styles.bl}>Put speakers at ease by making sure they're prepped and ready to go. Onboard them in the Green Room and communicate with them while they're on stage.</p>
```
Comment (joshi008) [inline components/AttendeesSpeakers/attendes.js]: Dont put single inverted commas ' . Instead use escape sequences.

**Review by joshi008 — CHANGES_REQUESTED** [review]
Try to remove these warnings as well:

image

File: components/AttendeesSpeakers/attendes.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+                        <h1 className={styles.h1}>A first-rate virtual experience for all</h1>
+                   
+                </div>
+                <div className={styles.grid}>
+                    <div className={styles.text}>
+                        <h4 className={styles.h4}>Attendees</h4>
+                        <p className={styles.bl}>No Zoom fatigue here. Optimized to create immersive experiences for attendees, Welcome captivates your audience like never before. Interact with attendees by pulling them on stage, allow them to move freely in and out of rooms, provide networking opportunities, and more.</p>
+                    </div>
+                    <div data-aos="fade-left" className={styles.img}>
+                        <img data-aos="rotate-c" src="https://assets-global.website-files.com/5f8f45cead04af2f54abc938/611fc7ce31ecb97d9ed140ab_use_attendee.jpg" className={styles.hi}/>
```
Comment (starwiz-7) [inline components/AttendeesSpeakers/attendes.js:26]: Use Image tag from next/Image. Add the domain name of image src to next.config.js to configure next/Image

File: styles/attendees.module.css [hunk 3]
```diff
@@ -0,0 +1,181 @@
+@import url('https://fonts.googleapis.com/css2?family=Martel+Sans:wght@400;700;800&display=swap');
```
Comment (starwiz-7) [inline styles/attendees.module.css]: Do not use any new font, kindly use the font variable available in the global CSS file.

**Comment by starwiz-7** [comment]
1. Isn't responsive for screens around 880 - 890px (Complete image isn't visible)
2.  Scale down the image, currently, the size of the image is very large(Depends on our posters)

**Review by starwiz-7 — APPROVED** [review]

**Review by starwiz-7 — APPROVED** [review]

**Pull request merged**